blocks/api/registration.js

```diff
@@ -330,3 +330,25 @@ export const getChildBlockNames = ( blockName ) => {
 export const hasChildBlocks = ( blockName ) => {
 	return select( 'core/blocks' ).hasChildBlocks( blockName );
 };
+
+/**
+ * Registers a new block style variation for the given block
+ *
+ * @param {string} blockName      Block type name.
```

Comment:
Block type name might actually be slightly confusing comment here because it implies blocks can have types and that’s what this is? That’s how I read it. It would be better to just say “name of block”. Maybe even provide an example like “Name of block (example: “latest-posts”).”